### PR TITLE
Bugfixes for c, rust

### PR DESCRIPTION
**Pull request by chrisnicollo**
Apologies, I accidentally closed the first pull request for this.

When these insert library functions were called in their respective talon files, the first argument was the name of the library, and the second argument was an empty string. As a result, it was pasting an empty string ("selection") instead of the library name ("text").

Also, one of the capture rules for c didn't match the name used for it in the associated talon file, causing the voice commands to not work.

### CODE REVIEW

File: lang/c/c.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def c_cast(m) -> str:
 
 
 @mod.capture(rule="[<self.stdint_signed>] <self.stdint_types> [<self.c_pointers>+]")
-def c_stdint_cast(m) -> str:
+def stdint_cast(m) -> str:
```
Comment (auscompgeek) [inline lang/c/c.py:191]: I wonder whether we should change the `.talon` file to match this instead.

**Review by auscompgeek — COMMENTED** [review]
The `code_insert_library` fixes LGTM.

File: lang/c/c.py (hunk 1 above)
Comment (chrisnicollo) [reply lang/c/c.py:191]: My reasoning for changing the ```.py``` file instead was that this happened to be the only place where the 'c_' prefix was used in front of a 'stdint' name. Changing the ```.talon``` file to match this one difference could potentially result in stylistic inconsistency.

**Review by splondike — APPROVED** [review]

**Comment by splondike** [comment]
Thanks!

**Pull request merged**